WordPress/Classes/ViewRelated/System/Coordinators/MySitesCoordinator.swift

```diff
@@ -131,15 +131,15 @@ class MySitesCoordinator: NSObject {
     func showBlogDetails(for blog: Blog, then subsection: BlogDetailsSubsection) {
         showBlogDetails(for: blog)
 
-        let blogDetailsViewController: BlogDetailsViewController?
-
         if Feature.enabled(.newNavBarAppearance) {
-            blogDetailsViewController = navigationController.topViewController?.children.first as? BlogDetailsViewController
+            if let mySiteViewController = navigationController.topViewController as? MySiteViewController {
```

Comment:
I think we could combine these ifs so we don't have the nesting – something like:

```
if Feature.enabled(.newNavBarAppearance),
    let mySiteViewController = navigationController.topViewController as? MySiteViewController {
    mySiteViewController.showBlogDetailsSubsection(subsection)
} else if let blogDetailsViewController = navigationController.topViewController as? BlogDetailsViewController {
    blogDetailsViewController.showDetailView(for: subsection)
}